packages/components/src/mobile/picker/index.android.js

```diff
@@ -55,7 +55,7 @@ export default class Picker extends Component {
 		const { options, leftAlign } = this.props;
 
 		return options.map( ( option ) => (
-			<View key={ `${ option.label }-${ option.value }` }>
+			<Fragment key={ `${ option.label }-${ option.value }` }>
```

Comment:
👋 @ceyhun , looks like [there's this PR](https://github.com/WordPress/gutenberg/pull/27326) that intentionally replaces the Fragment for a View. Do you think we should keep the View?

Reply:
In that PR the short syntax of Fragment was used like `<> </>`, which didn't support keys and maybe they were not aware of the [keyed Fragments](https://reactjs.org/docs/fragments.html#keyed-fragments). And also another issue that was solved in that PR, which was using `index` as a key and that was also in the commit in which this change was first introduced [here](https://github.com/WordPress/gutenberg/commit/d40168833531cbcc371c186864ebe3f91f44f14d#diff-9daa2ef947ebc2f02763111c4ffcac3a4f67c13f5c7ead89294ab24c10ed4a99). 
I think adding an extra `View` to the component tree shouldn't be necessary and `Fragment` should be fine.